src/Image/Darkroom/GdLib.php

```diff
@@ -35,6 +40,11 @@ public function process(string $file, array $options = []): array
         return $options;
     }
 
+    /**
+     * @param SimpleImage $image
```

Comment:
I didn't want put absolute path for `claviska\SimpleImage` external class of Kirby.